### DISCUSSION TITLE
Roadmap for mono android ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [Live-Charts/LiveCharts2#516] [Q&A]
How do you schedule your roadmap to release this package for Android ? Thank you.  (Yes it is Uno hence .NET5 isn't an option)

![image](https://user-images.githubusercontent.com/43340959/172335702-8547ecfc-e3d1-4be4-8a12-c91e24dc6c7e.png)

**Comment by beto-rodriguez** [comment]
This is already supported since beta-330
